bin/convert_owl_ncbo2owl.pl

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/perl -np
```

Comment:
This is for some special modifications that need to be made to Bioportal's RxNorm.